internal/args/unmarshal.go

```diff
@@ -61,13 +62,24 @@ var unmarshalFuncs = map[reflect.Type]UnmarshalFunc{
 
 	reflect.TypeOf((*time.Time)(nil)).Elem(): func(value string, dest interface{}) error {
 		// Handle absolute time
-		t, err := time.Parse(time.RFC3339, value)
-		if err != nil {
-			return err
+		absoluteTimeParsed, absoluteErr := time.Parse(time.RFC3339, value)
+		// Handle relative time
+		relativeTimeParsed, relativeErr := tparse.ParseNow(time.RFC3339, value)
```

Comment:
could the lib parse something like `2020-01-06T12:00:00-3h` ?

Reply:
We could but it is not guaranteed to work for all units: https://github.com/karrick/tparse#absoluteduration